Terminal.Gui/Core.cs

```diff
@@ -1445,8 +1535,9 @@ public override bool ProcessKey (KeyEvent keyEvent)
 				return true;
 
 			switch (keyEvent.Key) {
-			case Key.ControlC:
-				// TODO: stop current execution of this container
+			case Key.ControlQ:
```

Comment:
Oh, I forgot about this.

This is what we had been discussing with @tig on a different thread.   I forgot I had added this as a convenience.  Guess it was a bad call.

Reply:
I needed the generic Ctrl + C to copy and Ctrl + X to cut so the best alternative I saw was to change to Ctrl + Q which I think is a good alternative.